Cypress/cypress/pages/workflow.js

```diff
@@ -3,66 +3,72 @@
 //// ******************* Choose Agent Page ********************************
 
 Cypress.Commands.add("chooseAgent", (Agent) => {
-  cy.get('[type="radio"]').eq(Agent).check();
-  cy.get('[type="radio"]').eq(Agent).should("be.checked");
+	cy.get('[type="radio"]').eq(Agent).check();
+	cy.get('[type="radio"]').eq(Agent).should("be.checked");
 });
 
 //// *******************Choose Workflow Page *****************************
 
 Cypress.Commands.add("chooseWorkflow", (option, subOption) => {
-  cy.get("[data-cy=WorkflowsRadioGroup]").should("be.visible");
-  if (option == 0) {
-    cy.wait(180);
-    cy.get("[data-cy=PredefinedWorkflowsRadioButton] :radio").check();
-    cy.get("[data-cy=PredefinedWorkflowsRadioButton] :radio").should(
-      "be.checked"
-    );
-    cy.get("[data-cy=PredefinedWorkflowsRadioGroup]").should("be.visible");
-    cy.get("[data-cy=PredefinedWorkflowsRadioGroup]")
-      .children()
-      .eq(subOption)
-      .within(() => {
-        cy.get("[type=radio]").check();
-        cy.get("[type=radio]").should("be.checked");
-      });
-  } else if (option == 1) {
-    cy.get("[data-cy=templateWorkflowsRadioButton] :radio").check();
-    cy.get("[data-cy=templateWorkflowsRadioButton] :radio").should(
-      "be.checked"
-    );
-    cy.get("[data-cy=templateWorkflowsRadioGroup]").should("be.visible");
-    cy.get("[data-cy=templateWorkflowsRadioGroup]")
-      .children()
-      .eq(subOption)
-      .within(() => {
-        // cy.get("[type=radio]").check();
-        // cy.get("[type-radio]").should("be.checked");
-        cy.get("[type=radio]").click();
-      });
-  } else if (option == 2) {
-    cy.get("[data-cy=myHubsRadioButton] :radio").check();
-    cy.get("[data-cy=myHubsRadioButton] :radio").should("be.checked");
-    cy.get("[data-cy=myHubDropDown] ").click();
-    cy.get("[data-cy=hubOption]").eq(subOption).click();
-  } else if (option == 3) {
-    cy.get("[data-cy=uploadYAMLRadioButton] :radio").check();
-    cy.get("[data-cy=uploadYAMLRadioButton] :radio").should("be.checked");
-    cy.get("[data-cy=uploadYAMLInput]").should("be.visible");
-    cy.get("[data-cy=uploadYAMLInput] input").attachFile(
-      "sample-workflow.yaml"
-    );
-  }
+	cy.get("[data-cy=WorkflowsRadioGroup]").should("be.visible");
```

Comment:
These changes are because of prettier. Can you reset the config to match with the regular ones we use!